Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3
+
+RUN python3 -m venv .venv
+RUN cd /
+ENV source .venv/bin/activate
```

Comment:
I've commented out(removed) lines 3-5 and everything works fine. Better to try and talk if it fails at your machine